docs/core/install/includes/linux-install-60-apt.md

```diff
@@ -17,7 +17,7 @@ sudo apt-get update && \
 > [!IMPORTANT]
 > If you receive an error message similar to **Unable to locate package dotnet-sdk-6.0**, see the [troubleshooting](#troubleshooting) section.
 
-### Install the runtime
+### Install .NET 6 Runtime
```

Comment:
Since the text below talks about ASP.NET, should the title also be ASP.NET 6 Runtime?

Reply:
I only put a version here because it's used with the Ubuntu articles that have different instructions for .NET 6. I've modified the paragraph to be a bit more generic up front and then specify the ASP.NET runtime.